src/goto-programs/show_goto_functions.cpp

```diff
@@ -62,14 +62,24 @@ void show_goto_functions(
                      << (fun.second.body_available() ? ""
                                                      : ", body not available")
                      << " */";
+        msg.status() << messaget::eom;
       }
-
-      msg.status() << messaget::eom;
     }
     else
     {
-      goto_functions.output(ns, msg.status());
-      msg.status() << messaget::eom;
+      auto &out = msg.status();
```

Comment:
I think this just makes the code harder to read, but if it is used then it should be used consistently below.